### PR TITLE
A3 fixes

### PR DESCRIPTION
**Pull request by jreidinger**

### CODE REVIEW

File: test/stage1_test.rb [hunk 1]
```diff
@@ -51,17 +51,42 @@
...
+        allow(Yast::Arch).to receive(:architecture).and_return("ppc64")
 
-      subject.propose
+        object_double(
+          "Yast::BootStorage",
+          prep_partitions: ["/dev/sda1", "/dev/sdb1", "/dev/sdc1"],
+          detect_disks:    nil,
+          disk_with_boot_partition: "/dev/sdb"
+        ).as_stubbed_const
+
```
Comment (imobach) [inline test/stage1_test.rb]: Interesting :)

File: test/test_helper.rb [hunk 2]
```diff
@@ -68,6 +68,11 @@ def mock_disk_partition
 
 # stub udev mapping everywhere
 RSpec.configure do |config|
+  Yast.import "Storage"
+  Yast.import "BootStorage"
+  Yast.import "Bootloader"
+  require "bootloader/udev_mapping"
```
Comment (imobach) [inline test/test_helper.rb]: Is this needed for all tests?

**Comment by imobach** [comment]
LGTM

File: test/test_helper.rb (hunk 2 above)
Comment (jreidinger) [inline test/test_helper.rb]: as it is below mocked, we need it to have such constant defined

**Pull request merged**